app/waf/src/models/tenantassoc.model.ts

```diff
@@ -1,31 +1,53 @@
 import {Entity, model, property} from '@loopback/repository';
+import {CommonResponse, CommonCollectionResponse} from '.';
 
 @model()
-export class TenantAssociation extends Entity {
+export class AdcTenantAssociation extends Entity {
   @property({
     type: 'string',
     id: true,
     required: true,
   })
-  tenantId: string;
+  adcId: string;
 
   @property({
     type: 'string',
+    id: true,
     required: true,
   })
-  adcId: string;
+  tenantId: string;
 
-  @property({
-    type: 'date',
-  })
-  createdAt?: string;
+  constructor(data?: Partial<AdcTenantAssociation>) {
+    super(data);
+  }
+}
 
+@model()
+export class Tenant extends Entity {
```

Comment:
To my understanding, we will not store any things about tenant except an Id. 
Do you really need the model?

Reply:
model decorator does not impact anything. Let's remove it.

Reply:
My concern is do we need the Tenant class?

Reply:
Just hope to generate the API doc and response in a consistent way with others. If we do not define its property and schema, it will become hard to generate api doc at least.

Reply:
I mean there's no need to have the class `Tenant`. Because we only retain a tenant Id, which is only a string in other tables.

Reply:
Yes, but we need a class with property decorator to generate API doc. Can you please propose your idea?

Reply:
Tenant(the Id, of string type) is only a field of other tables, instead of a model/table, I think.

Reply:
Maybe you did not get what I mean...